widgetssdk/src/main/java/com/glia/widgets/callvisualizer/controller/VisitorCodeController.kt

```diff
@@ -20,6 +21,8 @@ class VisitorCodeController(
         dialogController.dismissVisitorCodeDialog()
     }
 
+    private val defaultRefreshDuration: Long = 180000L
```

Comment:
I would like to suggest specifying the unit.
For example:
`defaultRefreshDurationMilliseconds`
or add a comment `// X minutes` or whatever
Because currently it will take some time to figure out what 180000 actually is.

Reply:
I prefer to write the number as multipliers:
`3 * 60 * 1000`